### DISCUSSION TITLE
Automated import for GPX

### DISCUSSION THREAD

**Discussion by djhrum** [Freika/dawarich#70] [Ideas]
It would be good to have API endpoint to import GPX file. 
This endpoint allows to create simple watchdog on cron and bash to add newly added files in folder.

More enhanced solution: API call to scan initially configured folder and add all new files.

And third iteration: subscribe on system events or trigger second API endpoint by countdown.

**Comment by Freika** [comment]
Importing API is not a thing yet, but Dawarich can now monitor a folder to import files from there: https://github.com/Freika/dawarich/releases/tag/0.15.0
